### PR TITLE
Typereuse issue

### PR DESCRIPTION
**Pull request by ZhaodongTian**
Exclude the unavailable types from assembly

### CODE REVIEW

File: src/dotnet-svcutil/lib/src/CodeDomFixup/CodeDomVisitors/NamespaceFixer.cs [hunk 1]
```diff
@@ -21,7 +21,7 @@ internal class NamespaceFixup : CodeDomVisitor
         public NamespaceFixup()
         {
             // Microsoft.Xml.dll
-            var msxmlTypes = TypeLoader.LoadTypes(typeof(Microsoft.Xml.XmlDocument).GetTypeInfo().Assembly, Verbosity.Silent);
+            var msxmlTypes = TypeLoader.LoadTypes(typeof(Microsoft.Xml.XmlDocument).GetTypeInfo().Assembly, Verbosity.Silent, null, null);
```
Comment (HongGit) [inline src/dotnet-svcutil/lib/src/CodeDomFixup/CodeDomVisitors/NamespaceFixer.cs:24]: is this change necessary?

File: src/dotnet-svcutil/lib/src/CodeDomFixup/CodeDomVisitors/NamespaceFixer.cs [hunk 2]
```diff
@@ -31,7 +31,8 @@ public NamespaceFixup()
             }
 
             // Microsoft.CodeDom
-            var mscodedomTypes = TypeLoader.LoadTypes(typeof(Microsoft.CodeDom.CodeObject).GetTypeInfo().Assembly, Verbosity.Silent);
+            var mscodedomTypes = TypeLoader.LoadTypes(typeof(Microsoft.CodeDom.CodeObject).GetTypeInfo().Assembly, Verbosity.Silent, null, null);
```
Comment (HongGit) [inline src/dotnet-svcutil/lib/src/CodeDomFixup/CodeDomVisitors/NamespaceFixer.cs:34]: same here?

File: src/dotnet-svcutil/lib/src/CommandProcessorOptions.cs [hunk 3]
```diff
@@ -1025,6 +1025,6 @@ public string ToTelemetryString()
                 .Where(o => o.CanSerialize)
                 .Select(o => $"{o.Name}:[{OptionValueParser.GetTelemetryValue(o)}]")
                 .Aggregate((num, s) => num + ", " + s).ToString();
-        }
+        }
```
Comment (HongGit) [inline src/dotnet-svcutil/lib/src/CommandProcessorOptions.cs:1028]: nit, can you please remove the extra spaces?

File: src/dotnet-svcutil/lib/src/TypeLoader.cs [hunk 4]
```diff
@@ -32,20 +35,26 @@ static public Assembly LoadAssembly(string path)
...
+                listType.AddRange(Array.FindAll<Type>(rtle.Types, delegate (Type t)
+                {
+                    return t != null;
+                }));
+
+                //type.Module or type.Assembly could throw if multiple assembly with same name get referenced but only one version get restored.                
+                listType = listType.Except(GetUnAvailableTypes(listType)).ToList();
 
-                if (verbosity > Verbosity.Normal)
+                if (verbosity > Verbosity.Normal || context == OperationalContext.Infrastructure)
```
Comment (HongGit) [inline src/dotnet-svcutil/lib/src/TypeLoader.cs:57]: Why is this needed?

File: src/dotnet-svcutil/lib/src/CodeDomFixup/CodeDomVisitors/NamespaceFixer.cs (hunk 1 above)
Comment (ZhaodongTian) [reply src/dotnet-svcutil/lib/src/CodeDomFixup/CodeDomVisitors/NamespaceFixer.cs:24]: Yes, I added two parameters in LoadTypes method

File: src/dotnet-svcutil/lib/src/CodeDomFixup/CodeDomVisitors/NamespaceFixer.cs (hunk 2 above)
Comment (ZhaodongTian) [reply src/dotnet-svcutil/lib/src/CodeDomFixup/CodeDomVisitors/NamespaceFixer.cs:34]: Yes, I added two parameters in LoadTypes method

File: src/dotnet-svcutil/lib/src/TypeLoader.cs (hunk 4 above)
Comment (ZhaodongTian) [reply src/dotnet-svcutil/lib/src/TypeLoader.cs:57]: That means we generated proxy code from visualstudio. so if  created from visualstudio, we need to record all warnings

File: src/dotnet-svcutil/lib/src/CommandProcessorOptions.cs (hunk 3 above)
Comment (ZhaodongTian) [reply src/dotnet-svcutil/lib/src/CommandProcessorOptions.cs]: Done

**Review by HongGit — APPROVED** [review]
Please follow the other exception type.
